### PR TITLE
Do the necessary renamings in the automation dashboard project

### PR DESCRIPTION
**Pull request by Hiba-Alili**

### CODE REVIEW

File: app/templates_versions/common/navigation.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
                 <!-- beginSubviews-->
                 <li ui-sref-active="active">
-                    <a ui-sref="portal.subview1" style="background-color: #002d66"><i class="fa fa-desktop"></i> <span class="nav-label">Cloud automation</span> </a>
+                    <a ui-sref="portal.subview1" style="background-color: #002d66"><i class="fa fa-desktop"></i> <span class="nav-label">Service automation</span> </a>
                 </li>
                 <li ui-sref-active="active">
                     <a ui-sref="portal.subview2" style="background-color: #002d66"><i class="fa fa-desktop"></i> <span class="nav-label">Workflow automation</span> </a>
```
Comment (mboussaa) [inline app/templates_versions/common/navigation.html:59]: Still have Workflow automation

**Review by dianajlailaty — APPROVED** [review]

File: app/templates_versions/common/navigation.html (hunk 1 above)
Comment (mboussaa) [reply app/templates_versions/common/navigation.html:59]: maybe it was deactivated. I don't know where is this displayed

**Comment by mboussaa** [comment]
maybe take advantage of this PR to update some old naming in README.md

**Review by mboussaa — APPROVED** [review]

**Comment by medou-boushab** [comment]
You need to change the 392 in automation-dashboard/app/scripts/main.js

@Hiba-Alili

**Review by medou-boushab — APPROVED** [review]

File: app/templates_versions/community/subviews.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
             "workflow-execution/templates/catalog-view.html",
             "workflow-execution/templates/workflow-variables.html"
         ],
-        "notAvailablePage": "workflow-automation/page_not_available_wf_automation.html",
+        "notAvailablePage": "workflow-execution/page_not_available_wf_execution.html",
```
Comment (mboussaa) [inline app/templates_versions/community/subviews.json:14]: are u sure about this change?

**Comment by mboussaa** [comment]
LGTM

File: app/templates_versions/community/subviews.json (hunk 2 above)
Comment (Hiba-Alili) [reply app/templates_versions/community/subviews.json:14]: Yes, i created the file in https://bitbucket.org/activeeon/cloud-automation/pull-requests/302/

**Pull request merged**